modules/tinymce/src/plugins/table/main/ts/selection/TableSelection.ts

```diff
@@ -5,39 +5,36 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { Arr, Optional, Optionals } from '@ephox/katamari';
+import { Arr, Optionals } from '@ephox/katamari';
+import { Attribute, Compare, SugarElement, SugarElements, SelectorFind } from '@ephox/sugar';
 import { TableLookup } from '@ephox/snooker';
-import { SugarElement } from '@ephox/sugar';
-import Editor from 'tinymce/core/api/Editor';
-import * as CellOperations from '../queries/CellOperations';
-import * as Ephemera from './Ephemera';
-import { Selections } from './Selections';
+import { Selections, CellOpSelection } from '@ephox/darwin';
```

Comment:
One more out of order import, this should be above the katamari import (as d is before k).